ref-exchange/src/account_deposit.rs

```diff
@@ -345,7 +345,7 @@ impl Contract {
             lostfound.deposit(token_id, amount);
             self.accounts.insert(&self.owner_id, &lostfound.into());
         } else {
-            env::panic("ERR: non-whitelisted token can NOT deposit into lost-found.".as_bytes());
+            env::panic(ERR30_NON_WHITELIST_TOKEN.as_bytes());
```

Comment:
As this method is called in a callback, it is better to avoid panic, using a log! instead.